File: corpus/go/fixture_06.go
package fixture6

type PriceBook6 struct {
    Multiplier float64
}

func CalculateDiscount6(cartItems []float64, taxRate float64) float64 {
    totalAmount := 0.0
    for _, price := range cartItems {
        totalAmount += price
    }
    if totalAmount > 60.0 {
        return totalAmount * (1.0 + taxRate)
    }
    return totalAmount
}

File: corpus/javascript/fixture_06.js
export function calculateDiscount6(cartItems, taxRate) {
  const totalAmount = cartItems.reduce((sum, item) => sum + item.price * (item.qty ?? 1), 0);
  return totalAmount > 60 ? totalAmount * (1 + taxRate) : totalAmount;
}

export class PriceBook6 {
  lookupPrice(sku) { return sku.length + 6; }
}
